python/pyarrow/jvm.py

```diff
@@ -54,11 +54,11 @@ def _from_jvm_int_type(jvm_type):
 
     Parameters
     ----------
-    jvm_type: org.apache.arrow.vector.types.pojo.ArrowType$Int
+    jvm_type : org.apache.arrow.vector.types.pojo.ArrowType$Int
 
     Returns
     -------
-    typ: pyarrow.DataType
+    typ : pyarrow.DataType
```

Comment:
```suggestion
    typ: pyarrow.DataType
```

Reply:
>  The colon must be preceded by a space, or omitted if the type is absent.
Quoted from https://numpydoc.readthedocs.io/en/latest/format.html#docstring-standard and numpydoc [does check](https://github.com/numpy/numpydoc/blob/master/numpydoc/validate.py#L75-L76) this.

Reply:
Well, can you disable rule PR10 then? This is bad typography: see https://english.stackexchange.com/a/67403

Reply:
I see, but this belongs to the numpydoc format :)
Besides that we can only enforce the spaced version, so in order to enforce consistency I'd rather keep the rule enabled.

Reply:
Unless you have an automated way of doing the change this is gonna be a chore.
Personally, I think it's a waste of time to try to enforce this.

Reply:
It's indeed a numpydoc requirement (as far as I know, it will render incorrectly when not having the required spaces around it).

I also don't like this that much (although I am used to it now, it's in my muscle memory ..), but to relax this, that's a discussion to have in the numpydoc project I think